testplan/runnable/base.py

```diff
@@ -331,6 +310,29 @@ def exporters(self):
                 exporter.parent = self
         return self._exporters
 
+    @staticmethod
+    def get_default_exporters(config):
```

Comment:
This doesn't make a lot of sense to me: a static method but taking self.cfg as a parameter.